src/components/widgets-settings/organizations/organizations.tmpl.html

```diff
@@ -4,9 +4,9 @@ <h5 translate>impac.widget.settings.organizations.select_companies</h5>
   <div class="widget-lines-container">
     <div class="widget-line" ng-repeat="org in dashboardOrganizations">
       {{org.label}}
-      <i ng-show="organizationMode == 'multiple'" ng-class="isOrganizationSelected(org.uid) ? 'fa fa-toggle-on' : 'fa fa-toggle-off'" ng-click="toggleSelectOrganization(org.uid)" uib-tooltip="{{isOrganizationSelected(org.uid) ? 'disable' : 'enable'}}" tooltip-append-to-body="true" />
+      <i ng-show="mode == 'multiple'" ng-class="isOrganizationSelected(org.uid) ? 'fa fa-toggle-on' : 'fa fa-toggle-off'" ng-click="toggleSelectOrganization(org.uid)" uib-tooltip="{{isOrganizationSelected(org.uid) ? 'disable' : 'enable'}}" tooltip-append-to-body="true" />
```

Comment:
Last thing ;) can we add the methods you created to `$scope`, so they can be used in the view? E.g
```html
<i ng-show="$scope.isMultiMode()" ...>
<i ng-show="$scope.isSingleMode()" ...>
```

Reply:
Done